src/pages/home/Home.tsx

```diff
@@ -58,39 +78,80 @@ const useStyles = makeStyles(() => ({
     lineHeight: '18px',
     color: '#FFFFFF',
   },
-
   menu: {
     display: 'flex',
-    flexFlow: 'row wrap',
+    flexFlow: 'row-reverse',
     background: 'none',
     maxHeight: '80px',
+    [theme.breakpoints.down('sm')]: {
+      justifyContent: 'center',
+    },
+  },
+  intro: {
+    maxWidth: 640,
+    fontFamily: 'Poppins',
+    textAlign: 'justify',
+  },
+  introContainer: {
+    display: 'flex',
+    justifyContent: 'center',
+    marginTop: 32,
   },
 }));
 
+const Navigation = () => {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.menu}>
+      <Link to="/comparisons" className={classes.menuLink}>
+        CONTRIBUTE
+      </Link>
+      <a
+        className={classes.menuLink}
+        href="https://wiki.staging.tournesol.app"
+        target="_blank"
+        rel="noreferrer"
+      >
+        ABOUT
+      </a>
+      <Link className={classes.menuLink} to="/donate">
+        DONATE
+      </Link>
+    </div>
+  );
+};
+
 const Home = () => {
   const classes = useStyles();
 
   return (
     <div className={classes.root}>
-      <div className={classes.rectangle67}>
-        <img src="/svg/Watering.svg" className={classes.wateringSunflower} />
-        <div className={classes.menu}>
-          <Link to="/comparisons" className={classes.menuLink}>
-            CONTRIBUTE
-          </Link>
-          <a
-            className={classes.menuLink}
-            href="https://wiki.staging.tournesol.app"
-            target="_blank"
-            rel="noreferrer"
-          >
-            ABOUT
-          </a>
-          <Link className={classes.menuLink} to="/donate">
-            DONATE
-          </Link>
-        </div>
-      </div>
+      <Grid container className={classes.rectangle67}>
+        <Grid xs={12}>
+          <Navigation />
+        </Grid>
+        <Grid xs={12} md={6} className={classes.titleContainer}>
+          <Typography variant="h1" className={classes.title}>
+            Collaborative content Recommendations
+          </Typography>
+        </Grid>
+        <Grid xs={12} md={6} className={classes.imageContainer}>
+          <img src="/svg/Watering.svg" className={classes.wateringSunflower} />
+        </Grid>
+      </Grid>
+      <Grid container className={classes.content}>
+        <Grid xs={12} className={classes.introContainer}>
```

Comment:
This warning is visible in the console
> Warning: Failed prop type: The prop `xs` of `Grid` must be used on `item`.


`<Grid container>` should contain `<Grid item>`
```suggestion
      <Grid container className={classes.rectangle67}>
        <Grid item xs={12}>
          <Navigation />
        </Grid>
        <Grid item xs={12} md={6} className={classes.titleContainer}>
          <Typography variant="h1" className={classes.title}>
            Collaborative content Recommendations
          </Typography>
        </Grid>
        <Grid item xs={12} md={6} className={classes.imageContainer}>
          <img src="/svg/Watering.svg" className={classes.wateringSunflower} />
        </Grid>
      </Grid>
      <Grid container className={classes.content}>
        <Grid item xs={12} className={classes.introContainer}>
```